settei/presets/celery.py

```diff
@@ -46,7 +46,7 @@ class WorkerConfiguration(LoggingConfiguration):
         """The url of the broker used by Celery.  See also Celery's and
         Kombu's docs about broker urls:
 
-        http://docs.celeryproject.org/en/latest/configuration.html#broker-url
+        https://docs.celeryproject.org/en/master/userguide/configuration.html#std:setting-broker_url
```

Comment:
I think this link is better.
```suggestion
        https://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-broker_url
```